index.js

```diff
@@ -26,15 +26,15 @@ var plugins = [
   { register: require('./routes/static-pages.js') },
   { register: require('./routes/users.js') },
   { register: require('./routes/sessions.js') },
-  // { 
-  //   register: require('yar'),
-  //   options: {
-  //     cookieOptions: {
-  //       password: 'passwords',
-  //       isSecure: false
-  //     }
-  //   } 
-  // },
+  { 
+    register: require('yar'),
```

Comment:
The only way I could figure out how to make my page work was to comment this out - something about it was breaking my code. I kept getting bad cookie error.. I finally changed something somewhere else that I could include this back in!